docs/getting-started/index.md

```diff
@@ -21,7 +21,22 @@ If you are planning to contribute a new feature or bug fix to the Backstage proj
 
 ## Prerequisites
 
-This guide assumes a basic understanding of working on a Linux based operating system and have some experience with the terminal, specifically, these commands: `npm`, `yarn`.
+Before we dive in, its good to have an understanding of these concepts which are central to Backstage's
+design and functionality. Being an expert in each of these concepts is not
+necessary, however having a base understanding of each will make administering, developing,
+configuring, and operating Backstage easier.
+
+- CHANGELOG - https://keepachangelog.com
+- Docker - https://www.docker.com/
+- Monorepo - https://semaphoreci.com/blog/what-is-monorepo
+- Node.js - https://nodejs.org
+- React - https://reactjs.org
+- Semantic Versioning - https://semver.org
+- TypeScript - https://www.typescriptlang.org
+- YAML - https://yaml.org
+- Yarn - https://www.pluralsight.com/guides/yarn-a-package-manager-for-node-js
```

Comment:
Would it be worth layering some of these concepts in later? I know this is just a lift and shift, but this seems like a lot of prerequisites to getting started

Reply:
Good point, bit of an information overload 😆, I'm wondering actually if its even needed like this at all, maybe for example when talking about doing something with docker you can already just add it as a pre-requisite there, that would also narrow this down a lot.

Reply:
Maybe something like [c7856c4](https://github.com/backstage/backstage/pull/27672/commits/c7856c4633cc60bd0395888b0afae26506256561)? 🤔

Reply:
yup, looks good!